docs/docs/concepts/deployment.md

```diff
@@ -0,0 +1,84 @@
+# Deployment
+
+Once you build your LangGraph agent, you then have to think about deploying it.
+How you deploy your applications depends a bit on what it is and how you intend to use it.
+For simple applications, it can be pretty easy to deploy LangGraph.
+As your applications get more complicated (more nodes, longer running) or rely more on persistence (conversation memory, long-term memory, human-in-the-loop) then deployment can get trickier.
+In this guide, we go over a few of the considerations you will want to take into account when deploying more complicated LangGraph applications.
+
+> 📘 LangGraph Platform
+>
+> This guide covers concepts related to deployment in general. [LangGraph Platform](./langgraph_platform.md) is an opinionated way to deploy LangGraph agents built by the LangChain team, and handles all the issues raised here.
+
+## Simple deployment
+
+If your application is simple, then it is quite easy to deploy LangGraph agents - you can deploy them as you would any other Python application. For example, you could [FastAPI](https://fastapi.tiangolo.com/) (or an equivalent framework) to deploy your LangGraph application.
+
+```python
+# main.py
+
+from fastapi import FastAPI
+from your_agent_package import graph
+
+app = FastAPI()
+
+@app.get("/")
+async def root():
+    return graph.invoke({...})
+```
+
+For simple applications, this is sufficient and is not that different from deploying Python applications in general, and so we will not go into this in too much detail here.
+
+For more complicated LangGraph deployments, you will run into some issues. The rest of this guide will discuss those issues.
```

Comment:
The phrasing makes it sound like it's a problem with langgraph but all of these issues arise frequently in various applications